src/containers/pages/MDALite/plans/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,118 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/MDALiteReports/MDALitePlansList handles search correctly: search results props 1`] = `
+Array [
+  Object {
+    "jurisdiction_root_parent_ids": "[\\"bac41d87-b09d-4e76-be3b-3b6d07399891\\"]",
+    "plan_date": 1610064000000,
+    "plan_effective_period_end": 1640822400000,
+    "plan_effective_period_start": 1610064000000,
+    "plan_id": "6b485d4b-1043-40ed-ab8a-c64d24045ada",
+    "plan_intervention_type": "MDA-Lite",
+    "plan_name": "MDA-Lite-2021-01-08",
+    "plan_status": "active",
+    "plan_title": "MDA-Lite kenya 2021-01-08",
+  },
+]
+`;
+
+exports[`components/MDALiteReports/MDALitePlansList renders plan definition list correctly: GenericPlansList props 1`] = `
+Object {
+  "fetchPlans": [Function],
+  "history": Object {
```

Comment:
i am more of the opinion that we should use equality assertions for things like prop checks as opposed to snapshots.